lib/components/Expenses.js

```diff
@@ -0,0 +1,53 @@
+import React, { Component } from 'react'
```

Comment:
@chelletuerk Let's pull this file out and the lib/components/App.js file out. Or move them to public. Not sure if they are duplicates or not.